src/check-reserve-update-frequency/check-reserve-update-frequency.spec.js

```diff
@@ -1,12 +1,11 @@
 // required libraries
 const BigNumber = require('bignumber.js');
+const { createBlockEvent } = require('forta-agent');
 const {
-  Finding,
-  FindingSeverity,
-  FindingType,
-  createBlockEvent,
-} = require('forta-agent');
-const { provideHandleBlock, teardownProvider } = require('./check-reserve-update-frequency');
+  provideHandleBlock,
+  teardownProvider,
+  createAlert,
```

Comment:
you import `createAlert` but your agent never exports this in its `module.exports`

Reply:
`createAlert` added to exports.